apis/eventstream/index.yaml

```diff
@@ -158,6 +159,11 @@ get:
               value: |
                 event: data_column_sidecar
                 data: {"block_root": "0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2", "index": "1", "slot": "1", "kzg_commitments": ["0x1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505cc411d61252fb6cb3fa0017b679f8bb2305b26a285fa2737f175668d0dff91cc1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505"]}
+            execution_payload_available:
```

Comment:
I think we need one event stream for execution payload, bid and payload attestation

Reply:
@ensi321, i feel you on that. lmk if this is what you were thinking, https://github.com/ethereum/beacon-APIs/pull/552/commits/bcbb22b490027c6243544a1d96f3217ceee50f3d